config/settings.yml

```diff
@@ -31,3 +31,7 @@ import:
       directory: museum/records
       only: !ruby/regexp '/met_museum_records.csv/'
       traject_file: met_csv_config
+    numismatics:
+      directory: coins
+      traject_file: numismatic_csv_config
+      agg_provider: Numismatics Society
```

Comment:
Now that #163 is in, have a look how this was done on other providers - you need a little more hierarchy for these provided configurations.